docs/stdlib/datetime.rst

```diff
@@ -298,28 +294,29 @@ EdgeDB stores and outputs timezone-aware values in UTC.
         db> select <cal::local_time>'22:00' + <duration>'1 hour';
         {<cal::local_time>'23:00:00'}
 
-    Duration is a fixed number of seconds and microseconds and isn't
-    adjusted by timezone, length of month or anything else in datetime
-    calculations.
-
     See functions :eql:func:`to_duration`, and :eql:func:`to_str` and
     date/time :eql:op:`operators <dtminus>` for more ways of working with
     :eql:type:`duration`.
 
+    For more information regarding interacting with this type, see
+    :eql:func:`to_duration`, :eql:func:`to_str` and
+    :eql:op:`operators <dtminus>`.
+
 
 ----------
 
 
 .. eql:type:: cal::relative_duration
 
-    A type representing a span of time.
+    Represents a timespan relative to a given unit of time.
 
-    Unlike :eql:type:`std::duration` a ``relative_duration`` is not a precise
-    measurement because it uses 3 different units under the hood: months, days
-    and seconds. However not all months have the same number of days and not
-    all days have the same number of seconds. For example 2020 was a leap year
-    and had 366 days. Notice how the number of hours in each year below is
-    different.
+    Unlike :eql:type:`std::duration`, :eql:type:`cal::relative_duration`
+    is an inprecise form of measurement. Months, days and seconds are used
+    to declare the relation of the time's duration.
+
+    Note that the month specified share a correlation to their amount of days,
```

Comment:
```suggestion
    For example 2020 was a leap year and had 366 days. Notice how the number of
    hours in each year below is different:
```

The change is more difficult to read and parse. I prefer the more concrete style of the original here.